_posts/02-01-01-Code-Style-Guide.md

```diff
@@ -2,60 +2,43 @@
 anchor: code_style_guide
 ---
 
-# Code Style Guide {#code_style_guide_title}
+# কোড স্টাইল গাইড {#code_style_guide_title}
 
-The PHP community is large and diverse, composed of innumerable libraries, frameworks, and components. It is common for
-PHP developers to choose several of these and combine them into a single project. It is important that PHP code adhere
-(as close as possible) to a common code style to make it easy for developers to mix and match various libraries for
-their projects.
+পিএইচপি একটি বিশাল ও ভিন্ন ধরণের কমিউনিটি এবং এটি অসংখ্য লাইব্রেরি, ফ্রেমওয়ার্ক এবং অন্যান্য কম্পোনেন্টস দ্বারা গঠিত। পিএইচপি ডেভেলপাররা সাধারণত এইগুলোর মধ্যে কয়েকটি নির্বাচন করেন এবং সেগুলো একত্রিত করে একটি প্রজেক্টে ব্যবহার করেন। PHP কোডিং এর ক্ষেত্রে একটি সাধারণ কোড স্টাইল মেনে চলা (যতটুকু সম্ভব) খুবই গুরুত্বপূর্ণ যেন ডেভেলপারেরা তাদের প্রজেক্ট এর জন্য সহজেই বিভিন্ন লাইব্রেরি একত্রিত করে ও সাদৃশ্য বিবেচনা করে কাজ করতে পারেন।
```

Comment:
How about: 
"পিএইচপি কমিউনিটি বিশাল ও বৈচিত্রময়, যাতে আছে অসংখ্য লাইব্রেরি, ফ্রেমওয়ার্ক এবং অন্যান্য কম্পোনেন্ট।"
instead of -
"পিএইচপি একটি বিশাল ও ভিন্ন ধরণের কমিউনিটি এবং এটি অসংখ্য লাইব্রেরি, ফ্রেমওয়ার্ক এবং অন্যান্য কম্পোনেন্টস দ্বারা গঠিত"